xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/editors/dataeditors.css

```diff
@@ -344,4 +344,18 @@ div#xwikiobjects input[type="radio"], div#xwikiclassproperties input[type="radio
 #switch-xclass {
   text-align: right;
   margin: 4px 0 -16px;
-}
\ No newline at end of file
+}
+
+@media (max-width: 616px) {
+  #propname, #proptype {
+    display: block;
+  }
+  #add_xproperty {
+    background-image: none;
+  }
+  .property-name-label {
+    background: transparent url("/xwiki/resources/icons/silk/bullet_add.png") left center no-repeat; 
```

Comment:
$xwiki.getSkinFile

but anyway, I would adjust the main #add_xproperty instead of having something this specific inside a media query.